### DISCUSSION TITLE
Feature request: show component versions in dashboard

### DISCUSSION THREAD

**Discussion by mstade** [supabase/supabase#3856] [Feature Requests]
There's so much happening with Supabase these days that it's getting tricky to keep track of it all! This is a good thing, but what would make it even better is if we could have a page somewhere in the dashboard that gives us a view of what the versions of the various deployed server side components are. This is especially important as environments are upgraded, so we can tell whether we could use some newly released feature either from the Supabase team or the various component vendors. For example, the [new convenience features in Postgres 14 when working with JSON columns](https://www.postgresql.org/docs/14/datatype-json.html#JSONB-SUBSCRIPTING) look great, and [let's not forget the amazing work @steve-chavez keeps churning out](https://github.com/supabase/postgrest-js/issues/197#issuecomment-933900724), but it's hard to say whether these can be used unless we also know which version of Postgres, Postgrest, Gotrue etc. are deployed in any given environment.

What would be even better than version numbers is a link to a change log, so we could see what's actually in there. I presume this might be tricky to do at first, so as an MVP just a page with versions for each component would go a long way I think.

What'ya reckon?

**Comment by ff6347** [comment]
For Postgres it is easy `select version()`. I also get the version of Kong in the response headers but all other components are unclear to me so I also would 'vote' for this feature. This makes maintaining a local docker-compose version for testing easier.

Sidenote: Does the Go CLI give the user the possibility to pin the versions of used images or is this hardcoded and changes with every new release?

**Reply by ff6347** [reply]
Some related discussions 

https://github.com/supabase/supabase/discussions/3984

https://github.com/supabase/supabase/discussions/3989

https://github.com/supabase/supabase/discussions/3990

and many others https://github.com/supabase/supabase/discussions?discussions_q=version

**Reply by ff6347** [reply]
>  Does the Go CLI give the user the possibility to pin the versions of used images or is this hardcoded and changes with every new release?

Looks like they are hardcoded. Only the postgres version can currently be controlled via config.

https://github.com/supabase/cli/blob/ebdb28defda3d439f6cb62fac276b5dda1bbf759/internal/utils/utils.go#L42
